.github/scripts/parse_rpt.py

```diff
@@ -31,3 +32,33 @@
         raise ValueError("LVS failed!")
     else:
         print("LVS is clean!")
+
+if len(sys.argv) > 1 and sys.argv[1] == "sky130hd_temp_full":
+    result_filename = "work/prePEX_sim_result" # might remove from check or need to implement different logic later if sim outputs change due to downstream changes
+    sim_state_filename = "work/sim_state_file.txt"
+
+    with open(result_filename) as f2, open("../../../.github/scripts/expected_sim_outputs/prePEX_sim_result.txt") as f1:
```

Comment:
the `...../expected_sim_outputs/prePEX_sim_result.txt` is not added yet? Is this PR still in progress?

Reply:
It has been added here [https://github.com/idea-fasoc/OpenFASOC/pull/270/files#diff-b88c7420f91a0c9ff62c4eb29a049edd0f4ce38053c9539c6dab911338ec2645](url), in PR #270

Reply:
understood , thanks!